environment.yml

```diff
@@ -40,6 +40,7 @@ dependencies:
       - DFO-LS>=1.5.3  # dev, tests
       - Py-BOBYQA  # dev, tests
       - fides==0.7.4  # dev, tests
+      - pyensmallen
```

Comment:
Please add the comment `# dev, tests` so we know in which environments this is needed.